src/main/kotlin/calculator/model/Operator.kt

```diff
@@ -0,0 +1,43 @@
+package calculator.model
+
+enum class Operator(
+    val value: String,
+) {
+    PLUS("+"),
+    MINUS("-"),
+    MULTI("*"),
+    DIVIDE("/"),
+    ;
+
+    fun operate(first: Int, second: Int): Int {
+        return when (this) {
+            PLUS -> first + second
+            MINUS -> first - second
+            MULTI -> first * second
+            DIVIDE -> first / second
+        }
+    }
```

Comment:
열거타입을 만들었다면 각각의 인스턴스에 대한 계산로직의 책임은 Operator 상위 타입이 아닌 구현체(PLUS, MINUS)등이 담당하는게 적절하지 않을까요?
```suggestion
    fun operate(first: Int, second: Int): Int {
        return this.calculate(first, second)
    }
```

Reply:
말씀해주신대로 수정했더니 훨씬 좋은 코드가 되었습니다! 감사합니다 

```kotlin
enum class Operator(
    val value: String,
    val action: (Int, Int) -> Int
) {
    PLUS("+", { a, b -> a + b }),
    MINUS("-", { a, b -> a - b }),
    MULTI("*", { a, b -> a * b }),
    DIVIDE("/", { a, b -> a / b }),
    ;

    fun operate(first: Int, second: Int): Int {
        return this.action(first, second)
    }
```